src/market_mad/remotes/linuxcontainers/monitor

```diff
@@ -190,6 +196,35 @@ class LinuxContainersMarket
 
         str << "#{name}=\"#{val}\"\n"
     end
+
+    def vm_tmpl64
```

Comment:
This is static, move this to a class variable that initialize the base64 upon intialization